sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -661,6 +662,173 @@ public void SetLoadingTexture(Texture2D loadingTexture)
 			Options.loadingTexture = loadingTexture;
 			_mapVisualizer.SetLoadingTexture(loadingTexture);
 		}
+
+		#region Location Prefabs Methods
+
+		/// <summary>
+		/// Places a game object prefabs at the specified LatLon on the Map.
+		/// </summary>
+		/// <param name="prefab"> A Game Object Prefab.</param>
+		/// <param name="LatLon">A Vector2d(Latitude Longitude) object</param>
+		public void SpawnPrefabAtGeoLocation(GameObject prefab, Vector2d LatLon, Action<List<GameObject>> callback = null, bool scaleDownWithWorld = true, string locationItemName = "New Location")
+		{
+			var coordinateArray = new string[] { LatLon.x + ", " + LatLon.y };
+
+			PrefabItemOptions item = new PrefabItemOptions()
+			{
+				findByType = LocationPrefabFindBy.AddressOrLatLon,
+				prefabItemName = locationItemName,
+				spawnPrefabOptions = new SpawnPrefabOptions()
+				{
+					prefab = prefab,
+					scaleDownWithWorld = scaleDownWithWorld
+				},
+
+				coordinates = coordinateArray
+			};
+
+			if (callback != null)
+			{
+				item.OnAllPrefabsInstantiated += callback;
+			}
+
+			if(!_vectorData.LayerProperty.vectorSubLayers.Contains(item))
+			{
+				_vectorData.LocationPrefabsLayerProperties.locationPrefabList.Add(item);
+				_vectorData.AddVectorLayer(item);
+			}
+		}
+
+		/// <summary>
+		/// Places a game object prefabs all locations specified by the LatLon array.
+		/// </summary>
+		/// <param name="prefab"> A Game Object Prefab.</param>
+		/// <param name="LatLon">A Vector2d(Latitude Longitude) object</param>
+		public void SpawnPrefabAtGeoLocation(GameObject prefab, Vector2d[] LatLon, Action<List<GameObject>> callback = null, bool scaleDownWithWorld = true, string locationItemName = "New Location")
```

Comment:
These SpawnPrefabAt functions have a lot in common; any way we can reduce redundancy?